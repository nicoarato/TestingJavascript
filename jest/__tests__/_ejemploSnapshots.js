const cliente = {
    nombre: 'Nico22',
    balance: 500,
    tipo: 'Premium',

};

describe('Testing al cliente', () => {
    test('Es Nico', () => {
        expect(cliente).toMatchSnapshot();
    })
})