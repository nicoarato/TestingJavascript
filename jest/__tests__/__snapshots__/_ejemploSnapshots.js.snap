// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Nico 1`] = `
Object {
  "balance": 500,
  "nombre": "Nico22",
  "tipo": "Premium",
}
`;
